modules/priceFloors.ts

```diff
@@ -1150,13 +1160,28 @@ registerOrtbProcessor({type: REQUEST, name: 'extPrebidFloors', fn: setOrtbExtPre
 
 /**
  * Validate userIds config: must be an object with array values
+ * Also populates the validUserIdTierFields set with field names in the format "userId.tierName"
  */
 function validateUserIdsConfig(userIds: Record<string, unknown>): Record<string, unknown> {
   if (!userIds || typeof userIds !== 'object') return {};
+
+  // Clear the previous set of valid tier fields
+  validUserIdTierFields.clear();
+
   // Check if userIds is an object with array values
-  const invalidKey = Object.entries(userIds).find(([, value]) => !Array.isArray(value));
+  const invalidKey = Object.entries(userIds).find(([tierName, value]) => {
```

Comment:
can this be just `some` instead of find since we dont actually care about the `key`